util/find-doc-nits

```diff
@@ -444,6 +444,22 @@ sub check {
         check_section_location($id, $contents, "EXAMPLES", "SEE ALSO");
     }
 
+    # Make sure every link has a section.
+    while ( $contents =~ /L<([^>]*)>/g ) {
+        my $target = $1;
+        next if $target =~ /^https?:/;
+        next if $target =~ m@^/@;
+        next if $target =~ m@\([1357]\)@;
```

Comment:
The regexp should be `m@\([1357]\)$@` (ending $)

Reply:
fixed.